### DISCUSSION TITLE
Multiple interest rates on a CPN?

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#245] [General]
In Shareworks you can store multiples interest rates along with their effective dates on a convertible note.
E.g.
1% interest for the first year
1.5% interest for the second year
2% interest after that

The OCF format does not support this as there's only the one field for interest rate.

**Comment by JSv4** [comment]
Converted to issue #262
